tests/test_pykemon.py

```diff
@@ -8,207 +8,219 @@
+
+    def test_pokedex_resource(self):
+        base_test(self, "pokedex")
+
+    def test_version_resource(self):
+        base_test(self, "version")
+
+    def test_version_group_resource(self):
+        base_test(self, "version-group")
+
+    def test_item_resource(self):
+        base_test(self, "item")
+
+    def test_item_attribute_resource(self):
+        base_test(self, "item-attribute")
+
+    def test_item_category_resource(self):
+        base_test(self, "item-category")
+
+    def test_item_fling_effect_resource(self):
+        base_test(self, "item-fling-effect")
+
+    def test_item_pocket_resource(self):
+        base_test(self, "item-pocket")
+
+    def test_machine_resource(self):
+        base_test(self, "machine", "id")
+
+    def test_move_resource(self):
+        base_test(self, "move")
+
+    def test_move_ailment_resource(self):
+        base_test(self, "move-ailment")
+
+    def test_move_battle_style_resource(self):
+        base_test(self, "move-battle-style")
+
+    def test_move_category_resource(self):
+        base_test(self, "move-category")
+
+    def test_move_damage_class_resource(self):
+        base_test(self, "move-damage-class")
+
+    def test_move_learn_method_resource(self):
+        base_test(self, "move-learn-method")
+
+    def test_move_target_resource(self):
+        base_test(self, "move-target")
+
+    def test_location_resource(self):
+        base_test(self, "location")
+
+    def test_location_area_resource(self):
+        base_test(self, "location-area")
+
+    def test_pal_park_area_resource(self):
+        base_test(self, "pal-park-area")
+
+    def test_region_resource(self):
+        base_test(self, "region")
+
+    def test_ability_resource(self):
+        base_test(self, "ability")
+
+    def test_characteristic_resource(self):
+        base_test(self, "characteristic", "id")
+
+    def test_egg_group_resource(self):
+        base_test(self, "egg-group")
+
+    def test_gender_resource(self):
+        base_test(self, "gender")
+
+    def test_growth_rate_resource(self):
+        base_test(self, "growth-rate")
+
+    def test_nature_resource(self):
+        base_test(self, "nature")
+
+    def test_pokeathlon_stat_resource(self):
+        base_test(self, "pokeathlon-stat")
+
+    def test_pokemon_resource(self):
+        base_test(self, "pokemon")
+
+    def test_pokemon_color_resource(self):
+        base_test(self, "pokemon-color")
+
+    def test_pokemon_form_resource(self):
+        base_test(self, "pokemon-form")
+
+    def test_pokemon_habitat_resource(self):
+        base_test(self, "pokemon-habitat")
+
+    def test_pokemon_shape_resource(self):
+        base_test(self, "pokemon-shape")
+
+    def test_pokemon_species_resource(self):
+        base_test(self, "pokemon-species")
+
+    def test_stat_resource(self):
+        base_test(self, "stat")
+
+    def test_type_resource(self):
+        base_test(self, "type")
+
+    def test_language_resource(self):
```

Comment:
We tests everything works but we do not test things that don't work.

Can you add some tests that raise 404 errors for resources that are not found?